# Custom exceptions
class SequenceError(Exception):
    pass
